Hodge_Zach/assignments/ennumerable.rb

```diff
@@ -0,0 +1,13 @@
+module MyEnumerable
+    def my_each
+        for i in  0..self.length-1
```

Comment:
another option for the range is to do '0...self.length'.  Three dots makes the range exclusive of the last value so you wouldn't need the '-1'